docs/tutorials/getting_started.md

```diff
@@ -0,0 +1,135 @@
+# Getting started
+
+## Prerequisites
+
+1. Python installed on your system.
+2. Access to a microgrid system supported by the `frequenz.sdk` or you can use
+   the sandbox.
+3. Basic knowledge of microgrid concepts.
+4. Familiarity with
+   [Channels](https://frequenz-floss.github.io/frequenz-channels-python/latest/).
```

Comment:
Actually at some point I think all these steps should be links to other docs:

1. To the minimum requirements (or to the whole installation section, as pointed out in a follow-up comment)
2. To some docs explaining how to use the sandbox (and eventually how to setup or buy a plan or something to get a "microgrid system" working)
3. An introduction to microgrid concepts (https://github.com/frequenz-floss/frequenz-sdk-python/issues/456)
4. Channels introduction (https://github.com/frequenz-floss/frequenz-channels-python/issues/177)

Reply:
agreed
https://github.com/frequenz-floss/frequenz-sdk-python/issues/671